migrator/src/main/java/com/redhat/lightblue/migrator/AbstractController.java

```diff
@@ -89,6 +93,7 @@ public void unlock(String id) {
         } catch (Exception e) {
             LOGGER.error("Error unlocking {}",id,e);
         }
+	myLocks.remove(id);
```

Comment:
Is it possible to leak those locks? Maybe use finally when removing a lock from myLocks?